.gitignore

```diff
@@ -59,3 +59,6 @@ target/
 
 # Vagrant files
 .vagrant/
+
+# Ansible things
+vars/*.yml
```

Comment:
I changed this so that it ignores `.yml` files in `vars/`. That way people can't accidentally commit their settings.